src/components/Header/index.js

```diff
@@ -20,7 +20,7 @@ const Header = props => {
         <BeeLogo />
       </div>
       <div className={`${styles.buttonArea}`}>
-        <Button style="create">Create New Bounty</Button>
+        <Button style="primary">Create New Bounty</Button>
```

Comment:
much better than create :)

Reply:
Yeah we really have no need for a unique "create" btn. Looking to delete that component.